test/metabase/db/schema_migrations_test.clj

```diff
@@ -1915,3 +1915,38 @@
         (migrate! :down 49)
         (is (= #{(format "/block/db/%d/" db-id)}
                (t2/select-fn-set :object (t2/table-name :model/Permissions) :group_id group-id)))))))
+
+(deftest view-count-test
+  (testing "report_card.view_count and report_dashboard.view_count should be populated"
+    (impl/test-migrations ["v50.2024-04-25T16:29:31" "v50.2024-04-25T16:29:34"] [migrate!]
+      (let [user-id 13371338 ; use internal user to avoid creating a real user
```

Comment:
We hard code this in a few places. My "clean coder" cosmetic indoctrination is burning. What's the reason not to use `metabase.config/internal-mb-user-id`?

Reply:
it's unlikely, but it could change